### DISCUSSION TITLE
Where are MSV values in simulation?

### DISCUSSION THREAD

**Discussion by moPellowe** [Open-Systems-Pharmacology/Forum#1133] [Questions & Problems] [answered]
In MoBi, where are the MSV values located within a simulation? I have set the MSVs using the ospsuite R toolbox, so the building block does not contain the updated values. How do I navigate to the simulation MSVs?

**Comment by StephanSchaller** [comment, marked as answer]
Hi Moriah,

these are located on the compound level:
![image](https://user-images.githubusercontent.com/25477643/154532929-58504ea1-b43d-42a5-85ed-89cca30f6711.png)

hope this helps
Stephan

**Reply by moPellowe** [reply]
Thank you, @StephanSchaller . :)
